Got error when trying to push large files.

I initialized the repository and added the tracked extensions as explained by the docs. I added the files using the git command "git add images/", commited them with "git commit -m 'my comment bla bla bla'" and then tryed to push. I can't use "git lfs push origin master", but only "git-lfs push origin master".
This is the error i got :

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x10 pc=0x47b605]

goroutine 29 [running]:
github.com/github/git-lfs/lfs.doApiBatchRequest(0xc820514000, 0x13, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/rick/go/src/github.com/github/git-lfs/lfs/client.go:399 +0x95
github.com/github/git-lfs/lfs.Batch(0xc820112000, 0x64, 0x64, 0x889e30, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/rick/go/src/github.com/github/git-lfs/lfs/client.go:199 +0x639
github.com/github/git-lfs/lfs.(*TransferQueue).batchApiRoutine(0xc820354000)
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:205 +0x436
created by github.com/github/git-lfs/lfs.(*TransferQueue).run
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:328 +0x2b7

goroutine 1 [chan send]:
github.com/github/git-lfs/lfs.(*Batcher).Add(0xc82000f660, 0x7faa69326510, 0xc8203a4f80)
	/Users/rick/go/src/github.com/github/git-lfs/lfs/batcher.go:38 +0xeb
github.com/github/git-lfs/lfs.(*TransferQueue).Add(0xc820354000, 0x7faa69326510, 0xc8203a4f80)
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:71 +0xe0
github.com/github/git-lfs/commands.uploadPointers(0xc8200fa000, 0xa85, 0xa85, 0xc8200fa000)
	/Users/rick/go/src/github.com/github/git-lfs/commands/command_push.go:106 +0x725
github.com/github/git-lfs/commands.uploadsBetweenRefAndRemote(0x7ffe3fb8fa14, 0x6, 0xc82000ef50, 0x1, 0x1, 0x755540)
	/Users/rick/go/src/github.com/github/git-lfs/commands/command_push.go:73 +0x7be
github.com/github/git-lfs/commands.pushCommand(0xaaec60, 0xc82000ef40, 0x2, 0x2)
	/Users/rick/go/src/github.com/github/git-lfs/commands/command_push.go:197 +0x657
github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra.(*Command).execute(0xaaec60, 0xc82000ed60, 0x2, 0x2, 0x0, 0x0)
	/Users/rick/go/src/github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra/command.go:477 +0x403
github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra.(*Command).Execute(0xaafb00, 0x0, 0x0)
	/Users/rick/go/src/github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra/command.go:551 +0x46a
github.com/github/git-lfs/commands.Run()
	/Users/rick/go/src/github.com/github/git-lfs/commands/commands.go:88 +0x23
main.main()
	/Users/rick/go/src/github.com/github/git-lfs/git-lfs.go:9 +0x14

goroutine 23 [chan receive]:
github.com/github/git-lfs/lfs.(*TransferQueue).errorCollector(0xc820354000)
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:256 +0x5b
created by github.com/github/git-lfs/lfs.(*TransferQueue).run
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:317 +0x3f

goroutine 24 [chan receive]:
github.com/github/git-lfs/lfs.(*TransferQueue).retryCollector(0xc820354000)
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:263 +0x58
created by github.com/github/git-lfs/lfs.(*TransferQueue).run
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:318 +0x64

goroutine 25 [chan receive]:
github.com/github/git-lfs/lfs.(*TransferQueue).transferWorker(0xc820354000)
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:270 +0x98
created by github.com/github/git-lfs/lfs.(*TransferQueue).run
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:322 +0x167

goroutine 26 [chan receive]:
github.com/github/git-lfs/lfs.(*TransferQueue).transferWorker(0xc820354000)
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:270 +0x98
created by github.com/github/git-lfs/lfs.(*TransferQueue).run
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:322 +0x167

goroutine 27 [chan receive]:
github.com/github/git-lfs/lfs.(*TransferQueue).transferWorker(0xc820354000)
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:270 +0x98
created by github.com/github/git-lfs/lfs.(*TransferQueue).run
	/Users/rick/go/src/github.com/github/git-lfs/lfs/transfer_queue.go:322 +0x167

goroutine 28 [chan send]:
github.com/github/git-lfs/lfs.(*Batcher).acceptInput(0xc82000f660)
	/Users/rick/go/src/github.com/github/git-lfs/lfs/batcher.go:72 +0x12e
created by github.com/github/git-lfs/lfs.NewBatcher
	/Users/rick/go/src/github.com/github/git-lfs/lfs/batcher.go:26 +0xdd
```

--- a/lfs/client.go
+++ b/lfs/client.go
@@ -425,7 +425,7 @@ func doApiBatchRequest(req *http.Request) (*http.Response, []*objectResource, er
 	res, err := doAPIRequest(req, Config.PrivateAccess())
 
 	if err != nil {
-		if res.StatusCode == 401 {
+		if res != nil && res.StatusCode == 401 {
 			return res, nil, newAuthError(err)
 		}
 		return res, nil, err

